net/umurmur/files/umurmur.conf

```diff
@@ -0,0 +1,6 @@
+option umurmur 'config'
+	option conffile '/etc/umurmur/umurmur.conf'
+	option address4 '0.0.0.0'
+	option address6 '::'
+	option port4 '64738'
+	option port6 '64738'
```

Comment:
Doing this just for IP protocol and port does it make sense to introduce such breaking change?